frontend/src/metabase/admin/people/components/GroupSelect.jsx

```diff
@@ -17,7 +17,7 @@ import GroupSummary from "./GroupSummary";
 
 export const GroupSelect = ({
   groups,
-  selectedGroupIds = new Set(),
```

Comment:
Methods applied to `selectedGroupIds` down the line were always "includes".

We have been passing arrays to this props.

Once TypeScript reached a parent component of this one, the inconsistency beeped.